eagleproject/notify/actions/objects.py

```diff
@@ -148,3 +153,25 @@ def execute(self):
         else:
             # If we're not configured, don't die, but don't be silent
             log.info('Discord webhook is not configured')
+
+class Twilio(Action):
```

Comment:
We should discuss if we want to have alerts implemented both on analytics and on the new Prometheus stack. In my opinion I'd rather consolidate everything on the Prometheus side. Since it seems it would be less devops work for our team to have to maintain a single alerting system vs two.
For now I'm ok if we merge this as is but once Prometheus is fully rolled out and stable let's clean up.